[NPC] moved default dialogs out of NPC script and into DefaultActions for Mhaura

kaincenteno opened the pull request:
<!-- Remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm the following points: -->
<!-- (it should look like this: - [x] I have ...) -->
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

moved default dialogs out of NPC script and into DefaultActions for Mhaura
Also removed `local ID = require("scripts/zones/Mhaura/IDs")` as no IDs were being used in script

## Steps to test these changes

go to any npc in the list, and talk to them. Unless they are involved in a quest, they should return the default dialog.

kaincenteno committed:
added missing default dialog

WinterSolstice8 commented on scripts/zones/Mhaura/DefaultActions.lua:
```diff
@@ -1,12 +1,16 @@
+    ['Amalanbraux']      = { event = 700 },
+    ['Galdo']            = { event = 650 },
+    ['Hagain']           = { event = 10002 },
+    ['Itzha_Delavhitta'] = { event = 600 },
```
There are two event 600s? Is that right? Is this a pair of NPCs that responds with one event when clicking either?

TeoTwawki replied:
pretty sure old pxi and DSP scripts did that in a bunch of places to substitute for event ids they couldn't get or couldn't get working.

on occasion we've capped an event in retail and then watched the client NOT run the event due to some parameter being wrong or missing (ToAU had a few where we HAD to feed the event zeros for unused params, or they just would not load. And guild npc's require new functionality or would freeze)

kaincenteno replied:
> There are two event 600s? Is that right? Is this a pair of NPCs that responds with one event when clicking either?

yeah those default events were taken from the existing scipt (that was removed)

zach2good replied:
Sounds like we need a cap to see if these are both 600 with args, or if its a long standing bug

kaincenteno replied:
> Sounds like we need a cap to see if these are both 600 with args, or if its a long standing bug

got it. let me assign a soul in xiweb to get captures :D

kaincenteno replied:
@zach2good  daywhite from xiweb confirmed the CS already existing are accurate, they both use CS 600

WinterSolstice8 replied:
![image](https://user-images.githubusercontent.com/60417494/230818327-377c2b72-8410-42c9-af37-cd7f6e623bbd.png)

WinterSolstice8 replied:
![image](https://user-images.githubusercontent.com/60417494/230818431-aa7ab81b-0072-4ebd-a252-f2762c689e40.png)

WinterSolstice8 replied:
![image](https://user-images.githubusercontent.com/60417494/230818611-27cea88f-a0a0-45d4-be2c-7557f894c78c.png)

TeoTwawki replied:
_Teo cough's loudly_

kaincenteno replied:
thanks for checking @WinterSolstice8  :D  
i checked the recording again and you are right, she does CS 900, sorry for mistake @zach2good 

![image](https://user-images.githubusercontent.com/26943220/230821022-e33da5f4-c425-4829-b24a-ef7e2402de04.png)

@TeoTwawki  want a mint? :smile:

TeoTwawki replied:
> @TeoTwawki want a mint? 😄

\*lozenge

kaincenteno replied:
> > @TeoTwawki want a mint? smile
> 
> *lozenge

Ricolas are yummy :D

Made change to Itzha. Resolving convo

TeoTwawki replied:
a note for future reference: please let reviewers do the resolving.

kaincenteno replied:
oh crap i didnt know that, i have been doing it all along lol xD thanks for the note, Will make sure to do so in the future. unresolving it now. thanks @TeoTwawki  :D

TeoTwawki replied:
Don't sweat it. its githubs fault, been 6 months since I raised the issue of literally _anybody_ being able to resolve a convo and they didn't even acknowledge the issue report.

kaincenteno committed:
Itzha_Delavhitta default dialog is 900

WinterSolstice8 commented:
Could you also add 850 for Jikka-Abukka while you're here? thanks
![image](https://user-images.githubusercontent.com/60417494/232256726-a72bb62e-e121-438d-87ca-7fdd0758fa95.png)

kaincenteno committed:
moved  jikka-abukka to defaultactions

kaincenteno commented:
> Could you also add 850 for Jikka-Abukka while you're here? thanks ![image](https://user-images.githubusercontent.com/60417494/232256726-a72bb62e-e121-438d-87ca-7fdd0758fa95.png)

done, deleted their file since it only had the wrong trigger event and added the correct one to the defaultactions file.

TeoTwawki commented on scripts/zones/Mhaura/DefaultActions.lua:
```diff
@@ -1,12 +1,17 @@
+    ['Galdo']            = { event = 650 },
+    ['Hagain']           = { event = 10002 },
+    ['Itzha_Delavhitta'] = { event = 900 },
+    ['Jikka-Abukka']     = { event = 600 },
```
Jikka is 850, Lakom 600. so Jikka is still incorrect here.

kaincenteno committed:
jikka is 850
